frontend/src/components/UserTables/StudentsTable.tsx

```diff
@@ -86,18 +86,18 @@ const StudentsTable = ({ searchName }: studentTableProps) => {
         <Row header colSizes={colSizes} data={headers} />
 
         {(filter
-          ? riders.filter(
+          ? riders.filter((r) =>
+              (r.firstName + ' ' + r.lastName)
+                .toLowerCase()
+                .includes((searchName + '').toLowerCase())
```

Comment:
There's repeated logic here with the filtering, let's refactor that

Reply:
Got it. Sending a new request to patch this

Reply:
No need to make a new PR, you can make the changes to the current branch.